go.mod

```diff
@@ -41,20 +41,23 @@ require (
 )
 
 require (
+	github.com/flanksource/kommons v0.31.4
```

Comment:
Can we copy and paste the relevant code from kommons, we are trying to deprecate kommons